File: app/src/main/java/com/apps/straightface/gymstat/Data/Exercise.java
package com.apps.straightface.gymstat.Data;

import io.realm.RealmList;
import io.realm.RealmObject;


public class Exercise extends RealmObject
{
    private ExerciseObject exerciseType;
    private int numberOfSets;
    private RealmList<ExerciseEntry> exerciseEntries;

    public ExerciseObject getExerciseType()
    {
        return exerciseType;
    }
    public void setExerciseType(ExerciseObject exerciseType)
    {
        this.exerciseType = exerciseType;
    }

    public int getNumberOfSets()
    {
        return numberOfSets;
    }
    public void setNumberOfSets(int numberOfSets)
    {
        this.numberOfSets = numberOfSets;
    }

    public RealmList<ExerciseEntry> getExerciseEntries()
    {
        return exerciseEntries;
    }
    public void setExerciseEntries(RealmList<ExerciseEntry> exerciseEntries)
    {
        this.exerciseEntries = exerciseEntries;
    }
}

File: app/src/main/java/com/jorbital/gymstat/data/Exercise.kt
package com.jorbital.gymstat.data

import io.realm.RealmList
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

open class Exercise : RealmObject() {
    @PrimaryKey
    var idKey: String? = null
    var exerciseType: ExerciseObject? = null
    var numberOfSets: Int = 0
    var exerciseEntries: RealmList<ExerciseEntry> = RealmList()
    var connectedRoutine: String? = null
}
